io/src/lib.rs

```diff
@@ -52,6 +52,19 @@ pub trait Read {
     fn take(&mut self, limit: u64) -> Take<Self> { Take { reader: self, remaining: limit } }
 }
 
+/// A trait describing an input stream that uses an internal buffer when reading.
```

Comment:
I don't remember writing that, I've removed it and better described the PR. Thanks, attention to details like this is one of the reasons I like working in this project. Please always pick me up on anomalies like this, I try to write descriptive _correct_ commit logs that match the change because I think it makes for better patch sets and better reviews.

Reply:
I believe you wrote it before I told you how to fix the compilation issue - the CI was red at the time.

And if not for the bug I wouldn't like to block two-acked PR for something like this. Especially not *this PR*.